lib/avro/builder/aliasable.rb

```diff
@@ -0,0 +1,20 @@
+require 'avro/builder/dsl_attributes'
+
+module Avro
+  module Builder
+
+    # This is a shared concern for objects that support aliases via the DSL.
+    module Aliasable
+      def self.included(base)
+        base.dsl_attribute :aliases do |*names|
+          if !names.empty?
```

Comment:
We need the conditional because this is a combined reader/writer. If no argument is specified then we want to return the current value, not replace it.